### DISCUSSION TITLE
Minecraft crashing when viewing a recipe in REI

### DISCUSSION THREAD

**Discussion by War1412** [shedaniel/RoughlyEnoughItems#947] [Q & A] [answered]
Seems to be that whether I use R or ctrl + left click to display a recipe with REI, even on vanilla items, it saves my world and exits Minecraft

[Crash Log.txt](https://github.com/shedaniel/RoughlyEnoughItems/files/8955534/Crash.Log.txt)

**Comment by shedaniel** [comment, marked as answer]
Please look at #937, this is an issue by CC, and its poor JEI integration.
